### PR TITLE
lca - ibu: additionionalImages option is added.

### PR DESCRIPTION
**Pull request by achuzhoy**
This will allow providing a configmap reference with additional images.

### CODE REVIEW

File: pkg/lca/imagebasedupgrade.go [hunk 1]
```diff
@@ -223,6 +223,22 @@ func (builder *ImageBasedUpgradeBuilder) WithSeedImage(
...
 }
 
+// WithAdditionalImages adds additionalImages to be used by the imagebasedupgrade.
+func (builder *ImageBasedUpgradeBuilder) WithAdditionalImages(
+	additionalImagesConfigMapName, additionalImagesConfigMapNamespace string) *ImageBasedUpgradeBuilder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof("Appending additionalImages's configmap name %s in namespace %s to the imagebasedupgrade",
```
Comment (trewest) [inline pkg/lca/imagebasedupgrade.go:233]:
```suggestion
	glog.V(100).Infof("Appending additionalImages configmap name %s in namespace %s to the imagebasedupgrade",
```

**Review by trewest — CHANGES_REQUESTED** [review]
Looks good just one  nit

File: pkg/lca/imagebasedupgrade.go [hunk 2]
```diff
@@ -223,6 +223,22 @@ func (builder *ImageBasedUpgradeBuilder) WithSeedImage(
 	return builder
 }
 
+// WithAdditionalImages adds additionalImages to be used by the imagebasedupgrade.
+func (builder *ImageBasedUpgradeBuilder) WithAdditionalImages(
+	additionalImagesConfigMapName, additionalImagesConfigMapNamespace string) *ImageBasedUpgradeBuilder {
```
Comment (kononovn) [inline pkg/lca/imagebasedupgrade.go:228]: Should we validate that string parameters are not empty?

Comment (achuzhoy) [reply pkg/lca/imagebasedupgrade.go:228]: No need, patching with empty results in:

spec:
  additionalImages:
    name: ""
    namespace: ""

File: pkg/lca/imagebasedupgrade.go (hunk 1 above)
Comment (achuzhoy) [reply pkg/lca/imagebasedupgrade.go]: Updated. Thanks.

**Review by cdvultur — DISMISSED** [review]
/lgtm

File: pkg/lca/imagebasedupgrade.go [hunk 3]
```diff
@@ -223,6 +223,22 @@ func (builder *ImageBasedUpgradeBuilder) WithSeedImage(
...
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof("Appending additionalImages configmap name %s in namespace %s to the imagebasedupgrade",
```
Comment (trewest) [inline pkg/lca/imagebasedupgrade.go:233]: Just one technicality 
```suggestion
	glog.V(100).Infof("Adding additionalImages configmap name %s in namespace %s to the imagebasedupgrade",
```
Appending make it seem that it is part of a slice. But it is really just one configmap

Comment (achuzhoy) [reply pkg/lca/imagebasedupgrade.go]: You're right.
Thanks.

**Review by kononovn — APPROVED** [review]
/lgtm

**Pull request merged**